src/main/java/com/minecolonies/coremod/placementhandlers/PlacementError.java

```diff
@@ -0,0 +1,87 @@
+package com.minecolonies.coremod.placementhandlers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import net.minecraft.util.math.BlockPos;
+
+public class PlacementError {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)